packages/fluentui/react-bindings/src/hooks/useFrameListener.ts

```diff
@@ -0,0 +1,21 @@
+import * as React from 'react';
+import { FUI_FRAME_EVENT } from './useFrameFocusDispatch';
+
+export const useFrameListener = (callback: (e: Event) => void, $document: Document) => {
+  const listener = React.useCallback(
+    (e: Event) => {
+      if (callback) {
+        callback(e);
+      }
+    },
+    [callback],
+  );
+
+  React.useEffect(() => {
+    $document.addEventListener(FUI_FRAME_EVENT, listener);
```

Comment:
also check for `enable` here, the listener does not need to be added all the time